appveyor.yml

```diff
@@ -13,14 +13,37 @@ environment:
 
   matrix:
 
-      # We test Python 2.6 and 3.4 because 2.6 is most likely to have issues in
-      # Python 2 (if 2.6 passes, 2.7 virtually always passes) and Python 3.4 is
-      # the latest Python 3 release.
+#      - PYTHON_VERSION: "2.6"
+#        NUMPY_VERSION: "1.9"
+#        TEST_CMD: "import numpy; assert numpy.__version__.startswith(\"1.9\")"
 
-      - PYTHON_VERSION: "2.6"
-        NUMPY_VERSION: "1.9.1"
-      - PYTHON_VERSION: "3.4"
-        NUMPY_VERSION: "1.9.1"
+#      - PYTHON_VERSION: "3.4"
+#        NUMPY_VERSION: "1.9"
+#        TEST_CMD: "import numpy; assert numpy.__version__.startswith(\"1.9\")"
+
+#      - PYTHON_VERSION: "3.5"
+#        NUMPY_VERSION: "1.10"
+#        TEST_CMD: "import numpy; assert numpy.__version__.startswith(\"1.10\")"
+
+      - PYTHON_VERSION: "3.5"
+        NUMPY_VERSION: "1.10"
+        ASTROPY_VERSION: "development"
+        TEST_CMD: 'import numpy; assert numpy.__version__.startswith(str(1.10)); import astropy; assert astropy.__version__.startswith(\"1.1\")'
+
+      - PYTHON_VERSION: "3.5"
+        NUMPY_VERSION: "1.10"
+        ASTROPY_VERSION: "stable"
+        TEST_CMD: "import numpy; import astropy; assert numpy.__version__.startswith(\"1.10\"); assert astropy.__version__.startswith(\"1.0\");"
+
+      - PYTHON_VERSION: "3.5"
+        NUMPY_VERSION: "1.10"
+        ASTROPY_VERSION: "1.0"
+        TEST_CMD: 'import numpy; import astropy; assert numpy.all([numpy.__version__.startswith(\"1.10\"); astropy.__version__.startswith(\"1.0\")])'
+
+      - PYTHON_VERSION: "2.7"
+        NUMPY_VERSION: "1.10"
+        ASTROPY_VERSION: "1.0"
+        TEST_CMD: 'import numpy; import astropy; assert numpy.all([numpy.__version__.startswith(\"1.10\"), astropy.__version__.startswith(\"1.0\")])'
```

Comment:
I wonder if it might help debugging to split up the asserts into ``assert numpy.startswith(...); assert astropy.startswith(...)``? Maybe also then change ``\"`` back to ``"``?